.github/workflows/preview-commit.yml

```diff
@@ -0,0 +1,72 @@
+# https://github.com/stackblitz-labs/pkg.pr.new
+name: Preview Commit
+
+on:
+  workflow_call:
+
+jobs:
+  get-runner-labels:
+    name: Get Runner Labels
+    uses: ./.github/workflows/get-runner-labels.yml
+    with:
+      force-use-github-runner: true
+
+  build:
+    needs: [get-runner-labels]
+    strategy:
+      fail-fast: false # Build and test everything so we can look at all the errors
+      matrix:
+        array:
+          - target: x86_64-unknown-linux-gnu
+            runner: ${{ needs.get-runner-labels.outputs.LINUX_RUNNER_LABELS }}
+          - target: aarch64-unknown-linux-gnu
+            runner: ${{ needs.get-runner-labels.outputs.LINUX_RUNNER_LABELS }}
+          - target: x86_64-unknown-linux-musl
+            runner: ${{ needs.get-runner-labels.outputs.LINUX_RUNNER_LABELS }}
+          - target: aarch64-unknown-linux-musl
+            runner: ${{ needs.get-runner-labels.outputs.LINUX_RUNNER_LABELS }}
+          - target: i686-pc-windows-msvc
+            runner: ${{ needs.get-runner-labels.outputs.WINDOWS_RUNNER_LABELS }}
+          - target: x86_64-pc-windows-msvc
+            runner: ${{ needs.get-runner-labels.outputs.WINDOWS_RUNNER_LABELS }}
+          - target: aarch64-pc-windows-msvc
+            runner: ${{ needs.get-runner-labels.outputs.WINDOWS_RUNNER_LABELS }}
+          - target: x86_64-apple-darwin
+            runner: ${{ needs.get-runner-labels.outputs.MACOS_RUNNER_LABELS }}
+          - target: aarch64-apple-darwin
+            runner: ${{ needs.get-runner-labels.outputs.MACOS_RUNNER_LABELS }}
+    uses: ./.github/workflows/reusable-build.yml
+    with:
+      target: ${{ matrix.array.target }}
+      runner: ${{ matrix.array.runner }}
+      profile: "release-prod"
+      test: false
+
+  publish:
+    if: github.repository == 'web-infra-dev/rspack'
+    name: Pkg Preview
+    runs-on: ubuntu-latest
+    needs:
+      - build
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Pnpm Cache
+        uses: ./.github/actions/pnpm-cache
+
+      - name: Download artifacts
+        uses: actions/download-artifact@v4.1.7
+        with:
+          path: artifacts
+
+      - name: Build node packages
+        run: pnpm run build:js
+
+      - name: Move artifacts
+        run: node scripts/build-npm.cjs
+
+      - name: Show binding packages
+        run: ls -R npm
+
+      - name: Release
+        run: API_URL=https://pkg.pr.new pnpm dlx pkg-pr-new@https://7b32ad52.stackblitz-cr.pages.dev/stackblitz-labs/pkg.pr.new/playground-b@234 publish --compact --pnpm './npm/*' './crates/node_binding' './packages/rspack'
```

Comment:
maybe you should put API_URL in `env:`, but if it runs then no need

Reply:
Okay, but there are currently some issues with the assembly line. I will try again later.